omnibus/package-scripts/inspec/postinst

```diff
@@ -5,7 +5,7 @@
 #
 
 PROGNAME=`basename $0`
-INSTALLER_DIR=/opt/inspec/embedded
```

Comment:
To keep this consistent with other omnibus projects, I would leave this as `INSTALLER_DIR` and just change the value to `/opt/inspec`

Reply:
Thanks for the feedback @adamleff =)